coordinate_test.go

```diff
@@ -271,18 +278,21 @@ func TestCoordinate_parallelPings(t *testing.T) {
 			if err != nil {
 				r.Fatal(err)
 			}
-			expected := api.HealthChecks{
-				&api.HealthCheck{
-					Node:        node,
-					CheckID:     externalCheckName,
-					Name:        "External Node Status",
-					Status:      api.HealthPassing,
-					Output:      NodeAliveStatus,
-					CreateIndex: checks[0].CreateIndex,
-					ModifyIndex: checks[0].ModifyIndex,
-				},
+			expected := &api.HealthCheck{
+				Node:        node,
+				CheckID:     externalCheckName,
+				Name:        "External Node Status",
+				Status:      api.HealthPassing,
+				Output:      NodeAliveStatus,
+				CreateIndex: checks[0].CreateIndex,
+				ModifyIndex: checks[0].ModifyIndex,
+			}
+			if len(checks) != 1 {
+				t.Fatal("Bad number of checks; wanted 1, got ", len(checks))
```

Comment:
This should be `r.Fatal`, same with L294

Reply:
Thanks.